### DISCUSSION TITLE
Proxy SWA /api to other backends

### DISCUSSION THREAD

**Discussion by thomasgauvin** [Azure/static-web-apps-cli#559] [Ideas]
@manekinekko @sgollapudi77 @anthonychu @Reshmi-Sriram would it also be possible to support non-Azure Functions web apps? If I'm developing a SWA with an Express backend (for instance) and intend to deploy to App Service with /api integration, I should be able to develop locally with /api proxy to my express backend. Perhaps this should be spun out into it's own issue. Thoughts?

**Comment by thomasgauvin** [comment]
https://stackoverflow.com/questions/73765847/how-to-access-secured-api-management-apis-linked-to-an-azure-static-web-applicat

**Comment by TechWatching** [comment]
@thomasgauvin  did you manage to make the swa CLI work with a non Azure Functions api?
